src/content/configuration/resolve.md

```diff
@@ -115,6 +115,29 @@ The following table explains other cases:
 
 W> `resolve.alias` takes precedence over other module resolutions.
 
+`resolve.alias` now supports array of `strings` and `false` to ignore a module
+
+```js
+module.exports = {
+  //...
+  resolve: {
+    alias: {
+      _: [path.resolve(__dirname, 'src/utilities/'), path.resolve(__dirname, 'src/templates/')]
+    }
+  }
+};
+```
+
```

Comment:
Please add a description of what happens when you set it to `false`

Reply:
Would this work ?

`resolve.alias` when set to `false` will ignore a module.